src/abstractdataframe/show.jl

```diff
@@ -581,15 +581,23 @@ end
 showcols(df::AbstractDataFrame) = showcols(STDOUT, df) # -> Void
 
 using Juno
-using Juno: Inline, LazyTree, Table, Row, strong
+using Juno: Inline, LazyTree, Table, Row, strong, undefs
 
 const SIZE = 25
 
+function tomat(df::AbstractDataFrame)
```

Comment:
`convert(Matrix, ...)` also throws on `NA`s so this would involve a change of semantics – are you sure that's what you want?